mec/mec1501/component/espi_io.h

```diff
@@ -901,6 +901,122 @@ typedef struct espi_io_sirq_regs
 	__IOM uint8_t UART_2_SIRQ;
 } ESPI_IO_SIRQ_Type;
 
+/* All eSPI IO registers in one structure */
+struct espi_io_mbar { /* 80-bit register */
+	volatile uint16_t  LDN_MASK;
```

Comment:
drop the extra space after the type in these

Also, why use upper case for all of these? Is that some sort of convention?

Reply:
ARM CMSIS convention is upper case for register names in structures.

Reply:
Oh so that is where it came from. This is not a Zephyr convention though, right? That said, I see other vendors have done this in the main tree.

Reply:
no change